docs/pages/access-controls/reference.mdx

```diff
@@ -89,14 +89,20 @@ spec:
     max_sessions: 10
     # permit_x11_forwarding allows users to use X11 forwarding with openssh clients and servers through the proxy
     permit_x11_forwarding: true
-    # Specify whether or not to record the user's desktop sessions.
-    # Desktop session recording is enabled if one or more of the user's
-    # roles has enabled recording. Defaults to true if unspecified.
-    # Desktop sessions will never be recorded if auth_service.session_recording
-    # is set to 'off' in teleport.yaml or if the cluster's session_recording_config
-    # resource has set 'mode: off'.
+    # Session recording options.
     record_sessions:
+      # Specify whether or not to record the user's desktop sessions.
+      # Desktop session recording is enabled if one or more of the user's
+      # roles has enabled recording. Defaults to true if unspecified.
+      # Desktop sessions will never be recorded if auth_service.session_recording
+      # is set to 'off' in teleport.yaml or if the cluster's session_recording_config
```

Comment:
Which host's `teleport.yaml` would this be? The Auth Service?

Reply:
Yes, I think it is. I haven't changed this part, only moved. Do you think we need some improvements on it?

Reply:
I think it would help to make that more explicit, otherwise a user can mistake this for the wrong `teleport.yaml`.